src/Core/Models/Api/Request/Accounts/RegisterRequestModel.cs

```diff
@@ -65,31 +65,9 @@ public IEnumerable<ValidationResult> Validate(ValidationContext validationContex
     {
         if (Kdf.HasValue && KdfIterations.HasValue)
         {
-            switch (Kdf.Value)
-            {
-                case KdfType.PBKDF2_SHA256:
-                    if (KdfIterations.Value < 5000 || KdfIterations.Value > 2_000_000)
-                    {
-                        yield return new ValidationResult("KDF iterations must be between 5000 and 2000000.");
-                    }
-                    break;
-                case KdfType.Argon2id:
-                    if (!KdfIterations.HasValue || KdfIterations.Value < 0)
```

Comment:
@justindbaur  Seems you are losing this .HasValue test with the move to the centralized valiator.

Reply:
@kspearrin I did lose it but in both places that it was lost it was already guaranteed for `!KdfIterations.HasValue` to evaluate to `false` because we are inside a if that has `KdfIterations.HasValue` guaranteed to be true.

Reply:
I see now.